vm/src/vm.rs

```diff
@@ -620,20 +620,28 @@ impl VirtualMachine {
         })
     }
 
-    pub fn _lt(&mut self, a: &PyObjectRef, b: PyObjectRef) -> PyResult {
-        self.call_method(a, "__lt__", vec![b])
+    pub fn _lt(&mut self, a: PyObjectRef, b: PyObjectRef) -> PyResult {
+        self.call_or_unsupported(a, b, "__lt__", "__gt__", |vm, a, b| {
```

Comment:
The opposite of `<` is `>=`, right?

Reply:
No, because `call_or_unsupported` flips the operands. If you think about it, a > b and b < a are equivalent. a > b and b <= a are not.

Reply:
Oh, yes, you are right! Sorry for that!